docs/how-to-guides/add-conditional-control-to-a-flow.md

```diff
@@ -59,7 +59,7 @@ activate:
   ![bypassed_nodes](../media/how-to-guides/conditional-flow-with-activate/bypassed_nodes.png)
 
 
-    (1) If a node has activate config and the value of `activate.when` is equals to `activate.is`, it will be bypassed.
```

Comment:
We may also add a note here, if we want to enforce one node always run, we can set the activate config as `dummy is dummy`
![image](https://github.com/microsoft/promptflow/assets/3871260/3effd667-6da8-4f63-b2fa-72ba8981a83d)

Reply:
Updated.